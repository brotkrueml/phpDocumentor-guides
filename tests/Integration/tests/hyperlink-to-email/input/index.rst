Some file
=========

`link to email <git@github.com>`__
