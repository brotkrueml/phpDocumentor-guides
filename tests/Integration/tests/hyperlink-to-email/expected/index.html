<!DOCTYPE html>
<html lang="en">
<head>
    <title>Some file</title>

</head>
<body>
    <div class="section" id="some-file">
    <h1>Some file</h1>

            <p><a href="mailto:git@github.com">link to email</a></p>
    </div>

</body>
</html>
